### ISSUE TITLE
The focus moves from configuartion pane to app launcher in o365 suite nav in webpart

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#3686]
Requested priority: (Blocking, High, Normal, Low) :  : Blocked, accessibility issue

Products/sites affected: (if applicable): Webpart dev on Microsoft Tenant site Collection

#### Describe the issue:
As soon as the save button (which is the last control in the configuartion pane is clicked) the focus automatically shifts to app launcher in o365 suite nav. Focus ideally it should remain in configuration pane itself.

#### Actual behavior:

focus automatically shifts to app launcher in o365 suite nav.

#### Expected behavior:

Focus ideally it should remain in configuration pane itself.

**Comment by aditima** [comment]
@ssingh28 - You should be able to use the onDismissed() callback on the Panel component to set focus the way your app needs.
